docs/data.md

```diff
@@ -816,11 +816,13 @@ Extract the data for a dataset from an org and persist it to disk.
 
 `mapping` and either `sql_path` or `database_url` must be supplied.
 
-Example: :
-
+Example: 
+```console
     cci task run extract_dataset -o mapping datasets/qa/mapping.yml -o sql_path datasets/qa/data.sql --org qa
```

Comment:
Let's drop leading spaces since we're using fences:
```suggestion
cci task run extract_dataset -o mapping datasets/qa/mapping.yml -o sql_path datasets/qa/data.sql --org qa
```